### PR TITLE
* Upgrade to sailor 2.6.21

### PR DESCRIPTION
**Pull request by denyshld**
* Annual audit of the component code to check if it exposes a sensitive data in the logs
* Annual npm vulnerabilities audit

### CODE REVIEW

File: lib/actions/sendTemplate.js [hunk 1]
```diff
@@ -121,7 +114,7 @@ function getTemplateModel(cfg) {
 function getMetaModel(cfg) {
     const client = mandrill.createClient(cfg);
     const templateName = cfg.templateName;
-    console.log('About to retrieve info about template:', templateName);
+    logger.info('About to retrieve info about template:', templateName);
```
Comment (shulkaolka) [inline lib/actions/sendTemplate.js:117]: Why logger instead of this.logger?

File: lib/actions/sendTemplate.js [hunk 2]
```diff
@@ -112,7 +105,7 @@ function getTemplateModel(cfg) {
         result.forEach((it) => {
             model[it.name] = it.name;
         });
-        console.log('Found %s templates', result.length);
+        logger.info('Found %s templates', result.length);
```
Comment (shulkaolka) [inline lib/actions/sendTemplate.js:108]: Why logger instead of this.logger?

File: lib/actions/sendTemplate.js [hunk 3]
```diff
@@ -83,24 +92,8 @@ function createParams(msg, cfg) {
...
-    if (result.status === 'rejected' || result.status === 'invalid') {
-        throw new Error(JSON.stringify(result));
-    }
-
-    return result;
-}
-
 function getTemplateModel(cfg) {
-    console.log('About to retrieve available templates');
+    logger.info('About to retrieve available templates');
```
Comment (shulkaolka) [inline lib/actions/sendTemplate.js:96]: Why logger instead of this.logger?

File: verifyCredentials.js [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+    const that = this;
+    this.logger.info('About to verify given API key by retrieving user from Mandrill');
 
     const client = mandrill.createClient(credentials);
 
     return new Promise((resolve, reject) => {
         client.users.info({}, resolve, reject);
     }).then((result) => {
-        console.log('Your username: %s. API key is valid.', result.username);
+        that.logger.info('Your username: %s. API key is valid.');
```
Comment (shulkaolka) [inline verifyCredentials.js:12]:
```suggestion
        that.logger.info('API key is valid.');
```

**Review by shulkaolka — CHANGES_REQUESTED** [review]
Please, add
`"buildType":"docker"`
to component.json

File: lib/actions/sendTemplate.js (hunk 1 above)
Comment (denyshld) [reply lib/actions/sendTemplate.js:117]: It's metadata uploader. No difference here

File: lib/actions/sendTemplate.js (hunk 2 above)
Comment (denyshld) [reply lib/actions/sendTemplate.js:108]: It's metadata uploader. No difference here

File: lib/actions/sendTemplate.js (hunk 3 above)
Comment (denyshld) [reply lib/actions/sendTemplate.js:96]: It's metadata uploader. No difference here

File: verifyCredentials.js (hunk 4 above)
Comment (denyshld) [reply verifyCredentials.js]: Fixed

**Review by shulkaolka — APPROVED** [review]

**Pull request merged**